server/runtime_lua_nakama.go

```diff
@@ -1129,7 +1136,8 @@ func (n *RuntimeLuaNakamaModule) rsaSHA256Hash(l *lua.LState) int {
 		return 0
 	}
 
-	rsaPrivateKey, err := x509.ParsePKCS1PrivateKey([]byte(key))
+	block, _ := pem.Decode([]byte(key))
```

Comment:
We should observe the error here and return early if non-nil.

Reply:
The second returned value of `pem.Decode` is not an error, but the remaining input if it has multiple pem formatted blocks. I'll check if the returned `block` is nil instead and raise an error if it is.

Reply:
Excellent point, the check if it's `nil` is a good move then.